### ISSUE TITLE
Syntax highlighting contrast is not accessible

### ISSUE THREAD

**Issue by jen-reeve** [nesi/support-docs#1371]
Syntax highlighting in code blocks often causes the color contrast between the text and background to drop below accessible levels. Should be fixable via changes to some specific color options in `theme.css`
